api/system-certificate/read

```diff
@@ -61,6 +65,24 @@ if ($cmd eq 'list') {
     set_default('CommonName','NethServer');
     set_default('EmailAddress',"");
     set_default('SubjectAltName',$db->get_type('DomainName'));
+
+    # Output all know  domains/FQDN of the server
+    my @domains = ();
+
+    foreach my $d ($dbd->get_all()) {
+        my $domain = $d->key;
+        push @domains, $domain;
+    }
+    foreach my $h ($dbh->get_all_by_prop('type'=>'self')) {
+        my $domain = $h->key;
+        push @domains, $domain;
+    }
```

Comment:
I'm not sure that mail domains should be added to LE: usually the mail domain is not a FQDN.

Let's get a real example: `nethesis.it`.
The MX record points to `nethservice.nethesis.it`, which is the FQDN of the mail server.

Reply:
done